source/documentation/using_the_api/api_reference.md

```diff
@@ -225,19 +241,26 @@ Example response:
    },
 ```
 
-### `GET /entries`
+### <a name="get-entries">Get all entries from a register</a>
 
-Get all entries from the register. For example, all updates there have ever been to the `local-authority-eng` register.
+Endpoint: `GET /entries/`
 
-> Results from this API call are paginated. This call will return the first 100 entries from the first page of the register. Use `limit` to define the maximum number of entries you want and `start` to define the entry number you want to start from (in ascending order).
+Parameters: 
+* start (Optional): Collection page number. Defaults to ?.
```

Comment:
I presume it defaults to 1. If you do:

```
curl -I -H 'Accept: application/json' 'https://local-authority-eng.register.gov.uk/entries/'
```

You'll see a response such as:

```http
HTTP/1.1 200 OK
Content-Type: application/json
Link: <?start=101&limit=100>; rel="next"
```

So it suggests that to get the next page you need to start at 101.